tools/trap/HowToUseTrap.md

```diff
@@ -303,6 +303,7 @@ PC_value	 Symbol_address	  Symbol_name	File_name
 App Crash point is as follows:
 [ Caller - return address (LR) - of the function which has caused the crash ]
 
+App name          : app2
```

Comment:
Please change the usage. Now we need to pass the kernel elf file as an argument compulsorily. Currently, we can use any elf file according to the instructions above.
Example line 283 : $ python3 ramdumpParser.py -e ../../build/output/bin/app2_dbg -t ./log_file

Reply:
Done